source/crud/tests/v2/updateWithPipelines.yml

```diff
@@ -0,0 +1,82 @@
+data:
+    - { _id: 1, x: 1, y: 1, t: {u: {v: 1}} }
+    - { _id: 2, x: 2, y: 1 }
+
+minServerVersion: '4.1.11'
+
+collection_name: &collection_name "test"
+database_name: &database_name "crud-tests"
+
+tests:
+  -
+    description: "UpdateOne using pipelines"
+    operations:
+      -
+        name: "updateOne"
+        arguments:
+            filter: { y: 1 }
+            update: [ { $replaceRoot: { newRoot: "$t" } }, { $addFields: { foo: 1 } } ]
```

Comment:
Would it make sense to include a `result` assertion for UpdateOneResult fields as other `updateOne` tests?

Reply:
Added `result` assertion to the test.